client/src/components/forms/ResetPasswordForm.js

```diff
@@ -0,0 +1,77 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { reduxForm, Field } from 'redux-form'
+import _ from 'lodash'
```

Comment:
Include the function that you use from lodash. This is because when you import it like this webpack actually packs the whole lodash into the final bundle. While if you only use functionality that you need, it only ads that function to the final bundle.